spec/requests/users_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :request do
+  describe ':index' do
+    before(:each) { get root_path }
+
+    it('status should be :ok') do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it('rendered :index template') do
+      expect(response).to render_template(:index)
+    end
+  end
+
+  describe ':show' do
+    before(:each) do
+      @author = User.create(name: 'James', posts_counter: 0, photo: '', bio: '')
+      get user_path(@author)
+    end
+
+    it('status should be :ok') do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it('should render :show template') do
+      expect(response).to render_template(:show)
+    end
+
+    it('should assign User.find(params[:id]) to @user') do
+      expect(assigns(:user)).to eq(@author)
+    end
+  end
```

Comment:
- [ ] For this action please include a test for `If the response body includes correct placeholder text`.